src/styles/primeng-theme/components/_button.scss

```diff
@@ -210,7 +210,6 @@
 .ui-button-icon-only {
     .fa {
         margin-top: -0.5em;
-        margin-left: -0.65em;
```

Comment:
Isn't a way to do this without a set number?

Reply:
actually removed all fixed margins, and then compared on the app and it does not affects any icons